config/main.py

```diff
@@ -2272,20 +2272,20 @@ def breakout(ctx, interface_name, mode, verbose, force_remove_dependencies, load
                       remains unchanged to limit the traffic impact """
 
     click.secho("\nAfter running Logic to limit the impact", fg="cyan", underline=True)
-    matched_item = [intf for intf, speed in del_intf_dict.items() if intf in add_intf_dict.keys() and speed == add_intf_dict[intf]]
+    matched_item = [intf for intf, speed in list(del_intf_dict.items()) if intf in list(add_intf_dict.keys()) and speed == add_intf_dict[intf]]
 
     # Remove the interface which remains unchanged from both del_intf_dict and add_intf_dict
-    map(del_intf_dict.pop, matched_item)
-    map(add_intf_dict.pop, matched_item)
+    list(map(del_intf_dict.pop, matched_item))
+    list(map(add_intf_dict.pop, matched_item))
```

Comment:
Can we remove these lines?

Reply:
Good question. We should ask the original author. The `2to3` tool flagged these lines stating that we should consider using `for` loops here instead.